.eleventy.js

```diff
@@ -89,6 +89,16 @@ module.exports = (config) => {
     throw new Error('readTemplate tag: unexpectedly reached end of template include directories')
   })
 
+  config.addShortcode('readReactExample', (componentName, exampleName) => {
+    try {
+      const examplePath = require.resolve(`@moduk/frontend/src/react/${componentName}/__examples__/${exampleName}.tsx`)
+      const code = readFileSync(examplePath, 'utf8')
+      return prettier.format(code, { parser: 'typescript' })
+    } catch {
+      return ''
```

Comment:
As it's a short code the return value is coerced to a string, so this returns an empty string on error.